core/src/main/java/io/neow3j/contract/ContractParameter.java

```diff
@@ -248,10 +248,20 @@ public static ContractParameter hash256(String hashHexString) {
         return hash256(Numeric.hexStringToByteArray(hashHexString));
     }
 
+    /**
+     * Creates a hash256 parameter from the given hash.
+     *
+     * @param hash a 256-bit hash.
+     * @return the contract parameter.
+     */
+    public static ContractParameter hash256(Hash256 hash) {
+        return hash256(hash.toArray());
+    }
+
     /**
      * Creates a hash256 parameter from the given bytes.
      *
-     * @param hash bytes (possibly a 256-bit hash) in little-endian order.
```

Comment:
Here the mention of the endianness should not be removed because the parameter is a plain byte array.

Reply:
Yes, you're right, I'll change it back.